README.md

```diff
@@ -30,25 +51,230 @@ npm install --save ipfsd-ctl
 
 IPFS daemons are already easy to start and stop, but this module is here to do it from JavaScript itself.
 
+### Local node
+
 ```js
 // Start a disposable node, and get access to the api
-// print the node id, and kill the temporary daemon
-
-// IPFS_PATH will point to /tmp/ipfs_***** and will be
-// cleaned up when the process exits.
+// print the node id, and stop the temporary daemon
 
-var ipfsd = require('ipfsd-ctl')
+const controllerFactory = require('ipfsd-ctl')
+const daemonFactory = controllerFactory()
```

Comment:
I know this is the third time I bring this up, but I really like to keep it consistent, please use the following

```
const DaemonFactory = require('ipfsd-ctl')
const df = new DaemonFactory()
```

DaemonFactory is a class
df is an instance